.github/workflows/rector_ci.yaml

```diff
@@ -54,4 +54,4 @@ jobs:
                     author_email: tomas@getrector.org
                 env:
                     # to get push access
-                    token: ${{ secrets.ACCESS_TOKEN }}
+                    GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
This was a bug.

@samsonasik It might help with automated pushing to the repository

Reply:
@TomasVotruba Thank you

Reply:
It's a rough ugess, I have not tested it nor came across the issue yet.

If that happens again, please link the pull-request and don't send any more commits. I'll try to investigate it

Reply:
@TomasVotruba  , it happen again, see latest commit at https://github.com/rectorphp/rector/pull/4489

Reply:
@TomasVotruba commit https://github.com/rectorphp/rector/pull/4489/commits/271dec62f91750d443c369808f0bd843d653625b only have 1 running action (kodiakhq).